packages/ui/src/components/modals/TeachSessionModal.tsx

```diff
@@ -486,7 +486,6 @@ class TeachModal extends React.Component<Props, ComponentState> {
 
     @autobind
     renderSelectedActivity(activity: BB.Activity): (JSX.Element | null) {
```

Comment:
There isn't change here to mimic EditDialogModal because there isn't a train dialog to get the stubName chosen. It doesn't seem too be detrimental though because there are different use cases which doesn't rely on visualizing it in the activity.

When teaching you already know the stub chosen because you just chose it using the dropdown. When editing you are reviewing what happened which is why it was more important.